src/everest/linting/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""The everest.linting module offers linting functionality.
```

Comment:
Think these files should be removed? Looks like they were just deleted in: https://careers.peopleclick.eu.com/careerscp/client_statoil/int1068669248/jobDetails.do?functionName=getJobDetail&jobPostId=53729&localeCode=en-us

Reply:
You are right. Merge bug